README.md

```diff
@@ -5,7 +5,8 @@
 
 # nerdctl: Docker-compatible CLI for containerd
 
-![logo](docs/images/nerdctl.svg)
+![logo light mode](docs/images/nerdctl.svg#gh-light-mode-only)
+![logo dark mode](docs/images/nerdctl-white.svg#gh-dark-mode-only)
```

Comment:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

> The old method of specifying images based on the theme, by using a fragment appended to the URL (`#gh-dark-mode-only` or `#gh-light-mode-only`), is closing down and will be removed in favor of the new method described above.